views/guide.lavender

```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```

Comment:
@mowchan The spaces between if statements and else statements do not render. They just made the code more readable.